### ISSUE TITLE
List of differences to org-gcal

### ISSUE THREAD

**Issue by Fuco1** [pooruse/org-sgcal#1]
Hi! 

I'm also thinking of writing my own package for talking to google calendar, mostly because I use multiple calendars and I need a good workflow to copy events between them and manage "visibility" (i.e. what events to copy and what not).  `org-gcal` is not very flexible in this regard.

I'm planning to base my package on that codebase though for the basic API and OAuth is already taken care of.

Is your package for "public use" or is it too opinionated for me to consider it as an alternative base implementation?

**Comment by pooruse** [comment]
Yes , this project can be “publish use“ . I use this package for my daily tasks. If you don't mind there are still some bugs in this package, just take it. Hope this project can help you.
